main.py

```diff
@@ -1,12 +1,253 @@
-import requests
+from video import Video
+from customer import Customer
+from rental import Rental
+from constants import *
+from program import *
+from pprint import pprint
 
-URL = "http://127.0.0.1:5000"
-BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
+def run_cli(play=True):
+    start_program() # main()
 
-def main():
-    print("WELCOME TO RETRO VIDEO STORE")
-    pass
+    #initialize Customer API
+    customerApi = Customer()
+    videoApi = Video()
+    rentalApi = Rental()
 
+    print(INSTRUCTIONS)
+    options = show_options()
+    
+    while play==True:
+        print_little_stars()
+        choice = make_choice(options)
+        print_little_stars()
+        print(f"Your choice was {choice}: {options[choice]}")
+        print_little_stars() 
+
+        if choice == 0: # GET ALL VIDEOS  
+            all_videos = videoApi.list_videos()
+            pprint(all_videos)
+            print_little_stars()
+
+        elif choice == 1: # POST - CREATE VIDEO 
+            print_stars_centered()
+            title = input("Enter video title: ")
+            release_date = input("Enter release in YYYY-MM-DD format : ")
+            total_inventory = input("Enter video's total inventory: ")
+            print_stars_centered()
+            try:
+                response = videoApi.create_video(title, release_date, \
+                    total_inventory)
+                if response.status_code != 201:
+                    raise Exception("\n\nSomething is wrong with the information you entered")
+                else:
+                    print_stars_centered()
+                    print(f"\nVideo {response.json()} was created succesfully")
+                    print_stars_centered()
+            except Exception as e:
+                print_stars_centered()
+                print(f"There was an issue creating that video: {e}")
+                print_stars_centered()
```

Comment:
try/except works for edge case handling, but doesn't provide the user information about what went wrong. Consider handling `400` level responses differently than `404` level responses. Remember that your json response should capture some of the details of whether the request was invalid, or something was not found, and you can show this to the user.